flixel/system/FlxSound.hx

```diff
@@ -404,8 +415,9 @@ class FlxSound extends FlxBasic
 	 *                         paused when you call play(), it will continue playing from its current
 	 *                         position, NOT start again from the beginning.
 	 * @param   StartTime      At which point to start plaing the sound, in milliseconds
+	 * @param	EndTime			At which point to stop playing the sound, in milliseconds
 	 */
-	public function play(ForceRestart:Bool = false, StartTime:Float = 0.0):FlxSound
+	public function play(ForceRestart:Bool = false, StartTime:Float = 0.0, EndTime:Float = 0.0):FlxSound
```

Comment:
`EndTime == 0` here used to mean "set it to `length`", but now the behavior of `set_endTime()` is different.

Maybe it should be `?EndTime:Float`, and `if (EndTime == null)`, we set it to `length`?